Examples/Physics_applications/spacecraft_charging/inputs_test_rz_spacecraft_charging_picmi.py

```diff
@@ -98,8 +98,8 @@ def compute_virtual_charge_on_spacecraft():
     that WarpX thinks there should be on the spacecraft.
     """
     # Get global array for the whole domain (across MPI ranks)
-    phi = PhiFPWrapper(include_ghosts=False)[:, :]
-    rho = RhoFPWrapper(include_ghosts=False)[:, :]
+    phi = PhiFPWrapper()[:, :]
+    rho = RhoFPWrapper()[:, :]
```

Comment:
Just to make sure I understood this PR correctly, these are intentionally using `[:, :]`, instead of `[(), ()]`, because we want to exclude ghost cells?

Reply:
Yes, that is correct.

Reply:
Fancy :D Had to remind myself of that convention again by reading your doc :)
https://pyamrex.readthedocs.io/en/latest/usage/compute.html